new-cep.md

```diff
@@ -0,0 +1,94 @@
+# Conda Release Process Proposal
+
+| Title                | Conda Release Process                          |
+| -------------------- | ---------------------------------------------- |
+| Status               | Draft                                          |
+| Author(s)            | Dan Meador dan@anaconda.com>                |
+| Created              | April 19, 2023                                 |
+| Updated              | April 19, 2023                                 |
+| Discussion           | NA                                             |
+| Implementation       | NA                                             |
+
+## Abstract
+
+This proposal aims to establish a clear and efficient release process for conda (and other conda projects) by implementing a pilot and co-pilot system, ensuring a smooth transfer of responsibilities between team members and the continuous improvement of release quality. There will also be one "jump seat" available for those that want to sit in and observe the release process, but not be responsible for any of it.
```

Comment:
```suggestion
This proposal aims to establish a clear and efficient release process for conda (and other conda projects) by implementing a pilot and co-pilot system, ensuring a smooth transfer of responsibilities between release managers and the continuous improvement of release quality. There will also be one "jump seat" available for those that want to sit in and observe the release process, but not be responsible for any of it.
```

Reply:
@LtDan33 We were also thinking of the a QA member in a piece for release. Do we mention that as well